<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TracDetails</class>
 <widget class="QWidget" name="TracDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Severity</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Priority</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Component</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="severityCombo"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="priorityCombo"/>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="componentCombo"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Version</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Resolve as</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="versionCombo"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="statusCombo"/>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="resolutionCombo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
